packages/styling/src/styles/theme.ts

```diff
@@ -278,12 +278,15 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean, depCom
 
     errorText: !isInverted ? p.redDark : '#ff5f5f',
     warningText: !isInverted ? '#333333' : '#ffffff',
+    severeWarningText: !isInverted ? '#D83B01' : '#FCE100',
     successText: !isInverted ? '#107C10' : '#92c353',
-    errorBackground: !isInverted ? 'rgba(245, 135, 145, .2)' : 'rgba(232, 17, 35, .5)',
-    blockingBackground: !isInverted ? 'rgba(250, 65, 0, .2)' : 'rgba(234, 67, 0, .5)',
-    warningBackground: !isInverted ? 'rgba(255, 200, 10, .2)' : 'rgba(255, 251, 0, .6)',
+    infoBackground: !isInverted ? p.neutralLighter : '#323130',
```

Comment:
did @betrue-final-final sign off on these? this will cause these background colors to no longer have transparency

do we have screenshots for how this affects list highlighting with SharePoint's dark blue theme?